pkg/collector/influxdb_collector.go

```diff
@@ -111,20 +108,19 @@ type queryResult struct {
 
 // getValue returns the first result gathered from an InfluxDB instance.
 func (c *InfluxDBCollector) getValue() (resource.Quantity, error) {
-	res, err := c.influxDBClient.QueryCSV(context.Background(), c.query, c.org)
+	client := *c.influxDBClient
```

Comment:
Maybe we can just change `c.influxDBClient` to not be a pointer type if this is otherwise needed.